CMakeLists.txt

```diff
@@ -1622,8 +1622,13 @@ if(HEX_FILES_TO_MERGE)
   message(VERBOSE "Merging hex files: ${HEX_FILES_TO_MERGE}")
 endif()
 
-if(EMU_PLATFORM)
-  include(${ZEPHYR_BASE}/cmake/emu/${EMU_PLATFORM}.cmake)
+if(SUPPORTED_EMU_PLATFORMS)
+  list(GET SUPPORTED_EMU_PLATFORMS 0 default_emu)
+  add_custom_target(run DEPENDS run_${default_emu})
```

Comment:
Not directly related to this line, but didn't know where else to put the following comment.

We need something similar for the `debugserver`, but as not all emulators has the `debugserver` target today then it should probably be something along:
```
if(TARGET debugserver_${default_emu})
  add_custom_target(debugserver DEPENDS debugserver_${default_emu})
endif()
```

so that only if the default emulator supports debugserver then we create that target.

Reply:
Done. This also helped me spot that I was missing the change from `EMU_PLATFORM` to `SUPPORTED_EMU_PLATFORMS` in `cmake/flash/CMakeLists.txt`.